app/templates/docs/action.html

```diff
@@ -2,642 +2,927 @@
-                  <li><code class="docutils literal"><span class="pre">name</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">powerUps</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">prefs</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">premiumFeatures</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">products</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">url</span></code></li>
-                  <li><code class="docutils literal"><span class="pre">website</span></code></li>
-                </ul>
-              </li>
-            </ul>
-          </li>
-        </ul>
-      </li>
-    </ul>
-  </div>
-  <div class="section" id="put-1-actions-idaction">
-    <h2>PUT <span class="target" id="id1">/1/actions/[idAction]</span><a class="headerlink" href="#put-1-actions-idaction" title="Permalink to this headline"></a></h2>
-    <ul class="simple">
-      <li><strong>Required permissions:</strong> read</li>
-      <li><strong>Arguments</strong>
-        <ul>
-          <li><code class="docutils literal"><span class="pre">text</span></code> (optional)
-            <ul>
-              <li><strong>Valid Values:</strong> a string with a length from <code class="docutils literal"><span class="pre">1</span></code> to <code class="docutils literal"><span class="pre">16384</span></code></li>
-            </ul>
-          </li>
-        </ul>
-      </li>
-    </ul>
-  </div>
-  <div class="section" id="put-1-actions-idaction-text">
-    <h2>PUT <span class="target" id="actions-idaction-text">/1/actions/[idAction]/text</span><a class="headerlink" href="#put-1-actions-idaction-text" title="Permalink to this headline"></a></h2>
-    <ul class="simple">
-      <li><strong>Required permissions:</strong> write</li>
-      <li><strong>Arguments</strong>
-        <ul>
-          <li><code class="docutils literal"><span class="pre">value</span></code> (required)
-            <ul>
-              <li><strong>Valid Values:</strong> a string with a length from <code class="docutils literal"><span class="pre">1</span></code> to <code class="docutils literal"><span class="pre">16384</span></code></li>
-            </ul>
-          </li>
-        </ul>
-      </li>
-    </ul>
-  </div>
-  <div class="section" id="delete-1-actions-idaction">
-    <h2>DELETE <span class="target" id="id2">/1/actions/[idAction]</span><a class="headerlink" href="#delete-1-actions-idaction" title="Permalink to this headline"></a></h2>
-    <ul class="simple">
+<div class="section" id="get-1-actions-idaction">
+  <h2>GET <span class="target" id="actions-idaction">/1/actions/[idAction]</span><a class="headerlink" href="#get-1-actions-idaction" title="Permalink to this headline"></a></h2>
+  <ul class="simple">
+    <li>
+      <strong>Examples</strong>
```

Comment:
@bentleycook Usually for our other docs, `Examples` comes last.

I think usually its `Required Permissions`, `Arguments`, and then occassionally, `Examples`